pkg/metrics/shoot.go

```diff
@@ -283,6 +283,18 @@ func (c gardenMetricsCollector) collectShootNodeMetrics(shoot *gardenv1beta1.Sho
 		nodeCountMax += worker.Minimum
 		nodeCountMin += worker.Maximum
 
+		var criName string
+		var containerRuntimes []string
+
+		if worker.CRI == nil {
+			criName = "docker (default)"
```

Comment:
Instead of using `docker (default)` here, can we just use `(default)`? If I understand correctly after k8s 1.22 the default will be containerd but the metric would still say `docker (default)` which is incorrect.

Reply:
As discussed just now: Shoots with k8s >= 1.22 will always have a value for `.spec.provider.workers[].cri.name`. The defaulting logic we added with https://github.com/gardener/gardener/pull/4222 takes care of the case where the user didn't specify anything.